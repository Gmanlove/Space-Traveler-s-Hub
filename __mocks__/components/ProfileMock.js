import React from 'react';
import { useSelector } from 'react-redux';

const ProfileMock = () => {
  const { reserveState } = useSelector(
    (store) => store.rocket,
  );
  const { missionReserve } = useSelector(
    (store) => store.mission,
  );

  return (
    <div className="Profile-holder">
      <h2>My Profile</h2>
      <div className="main">
        <div className="Missions">
          <h4>My Missions</h4>
          <div className="list">
            {missionReserve.map((mission) => (
              <p key={mission.id}>{mission.mission}</p>
            ))}
          </div>
        </div>
        <div className="Rockets">
          <h4>My Rockets</h4>
          <div className="list">
            {reserveState.map((reserved) => (
              <p key={reserved.id}>{reserved.title}</p>
            ))}
          </div>
        </div>
      </div>
    </div>
  );
};
export default ProfileMock;
