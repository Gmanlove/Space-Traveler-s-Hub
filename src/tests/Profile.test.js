import ProfileMock from '../../__mocks__/components/ProfileMock';

describe('Testing the profile', () => {
  const component = <ProfileMock />;

  test('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
