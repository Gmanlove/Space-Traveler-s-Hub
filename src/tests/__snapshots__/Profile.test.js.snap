// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the profile should match snapshot 1`] = `<ProfileMock />`;
